airflow/executors/executor_loader.py

```diff
@@ -35,6 +34,9 @@
 
 log = logging.getLogger(__name__)
 
+if TYPE_CHECKING:
+    from airflow.executors.base_executor import BaseExecutor
```

Comment:
By moving this here it prevents loading all the models and saves a goodly chunk of time.

Reply:
Fantastic!